.travis.yml

```diff
@@ -14,8 +14,10 @@ addons:
     - g++-4.8
     - clang
 
+env:
+  - CXX=g++-4.8 CC=gcc-4.8
+
 install:
-  - if [ "$CXX" = "g++" ]; then export CXX="g++-4.8" CC="gcc-4.8"; fi
```

Comment:
this is used for clang.

Reply:
no problem, sorry about the spurious pr!